man/leapp.1

```diff
@@ -0,0 +1,34 @@
+.TH LEAPP "1" "30/09/2018" "leapp 0.3" "User Commands"
+.SH NAME
+leapp - command-line interface to for App & OS Modernization Framework Leapp
+.SH SYNOPSIS
+leapp [-h|--help] [--version] [--debug] <command>
+.SH DESCRIPTION
+The leapp tool is an end-user application designed to run specific workflows.
+.SH OPTIONS
+-h, --help
+    show this help message and exit
+
+--version
+    show program's version number and exit
+
```

Comment:
I am missing the --whitelist-experimental option to be documented

Reply:
Well thinking of it there is more that is missing, the manpage from this PR olny covers the snactor command, common options and a list snactor commands. Command specific options are missing, e.g. `--whitelist-experimental` for `run` command, `--json` for `discover` etc.

The question is how to deal with it - either list these options in the `SNACTOR COMMANDS` section of man or use the approach similar to what `git` has, i.e. `man snactor` would print the common options and list of subcommands and then there would be `man snactor <subcommand>` manpage for each subcommand of the snactor tool.